index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div>
+        <h1 id="title">Microverse Survey Form</h1>
+        <p id="description">
+            Thank you for taking the time to help us improve our skills
+        </p>
+    </div>
+
+    <form id="survey-form">
+        <div>
+            <label id="name-label" for="name">Name</label>
+            <input type="text" name="name" id="name" placeholder="Enter your name" required="" />
+        </div>
+        <div>
+            <label id="email-label" for="email">Email</label>
+            <input type="email" name="email" id="email" placeholder="Enter your Email" required="" />
+        </div>
+
+        <div>
+            <label id="number-label" for="number">Age<span class="clue">(optional)</span></label>
+            <input type="number" name="age" id="number" min="10" max="99" placeholder="Age" />
+        </div>
+        <div>
+            <p>Which option best describes your current position?</p>
```

Comment:
**Finally**
I notice that you don't use a lot of tags and trust me there are a lot that you can use in your code to make it look more professional